lib/ibm_watson/base_service.rb

```diff
@@ -14,9 +14,10 @@ def version(version)
       end
     end
 
-    def initialize(username:, password:)
+    def initialize(username:, password:, timeouts: [:global, {write: 2, connect: 5, read: 5}])
```

Comment:
How about a comment linking to the documentation of the param? It's basically a straight through to http-rb.